test/Ocelot.AcceptanceTests/AggregateTests.cs

```diff
@@ -30,99 +30,101 @@ public AggregateTests()
 
         [Fact]
         [Trait("Issue", "597")]
-        public void should_fix_issue_597()
+        public void Should_fix_issue_597()
         {
             var port = PortFinder.GetRandomPort();
             var configuration = new FileConfiguration
             {
-                Routes = new List<FileRoute>
-                {
-                    new()
+                Routes =
+                [
+                    new FileRoute
                     {
                         DownstreamPathTemplate = "/api/values?MailId={userid}",
                         UpstreamPathTemplate = "/key1data/{userid}",
-                        UpstreamHttpMethod = new List<string> {"Get"},
+                        UpstreamHttpMethod = ["Get"],
                         DownstreamScheme = "http",
-                        DownstreamHostAndPorts = new List<FileHostAndPort>
-                        {
-                            new()
+                        DownstreamHostAndPorts =
+                        [
+                            new FileHostAndPort
                             {
                                 Host = "localhost",
                                 Port = port,
                             },
-                        },
+                        ],
                         Key = "key1",
                     },
```

Comment:
Just suggestion! You can use new `GivenRoute` helper.
```c#
private static FileRoute GivenRoute(int port, string upstream, string key);
```
But you need downstream argument, so you could redefine this method like this:
```c#
private static FileRoute GivenRoute(int port, string key, string upstream = null, string downstream = null);
```

Reply:
ok

Reply:
Not your code!
Let's forget about it!